workflow/rules/common.smk

```diff
@@ -1041,6 +1041,25 @@ def get_lineage_by_accession(wildcards):
     ]
 
 
+def get_include_flag_for_date(date):
```

Comment:
From some mistakes, we learned, that it is best to hand over the complete wildcards object. This makes it easier to include the functions down the road in some other functions more easily.